_examples/http-producer/main.go

```diff
@@ -0,0 +1,141 @@
+package main
```

Comment:
it is outdated? if not that some readme will be useful

Reply:
It's outdated, I've removed it altogether